### ISSUE TITLE
When I use 'onUpdate:value={doUpdate}' in tsx file, eslint reports an error

### ISSUE THREAD

**Issue by brenner8023** [vuejs/vue-eslint-parser#114]
my options
```js
module.exports = {
  parser: 'vue-eslint-parser',
  parserOptions: {
    parser: '@typescript-eslint/parser',
    sourceType: 'module',
    ecmaVersion: 6,
    ecmaFeatures: {
      jsx: true,
      tsx: true
    }
  },
  env: {
    browser: true,
    node: true,
    es6: true
  },
  plugins: ['@typescript-eslint'],
  extends: [
    'plugin:@typescript-eslint/recommended',
    'plugin:vue/vue3-recommended'
  ],
  rules: {
    quotes: ['error', 'single', { avoidEscape: true, allowTemplateLiterals: true }],
    'vue/max-attributes-per-line': 'off',
    'vue/no-multiple-template-root': 'off',
    '@typescript-eslint/no-explicit-any': 'off',
    '@typescript-eslint/member-delimiter-style': [
      'error',
      {
        multiline: {
          delimiter: 'none',
          requireLast: false,
        },
        singleline: {
          delimiter: 'semi',
          requireLast: true,
        },
      },
    ]
  }
};
```

my code:
![image](https://user-images.githubusercontent.com/31237954/124490854-2c92ce80-dde5-11eb-8c43-a4a580070432.png)

![image](https://user-images.githubusercontent.com/31237954/124491408-d3776a80-dde5-11eb-8119-db41dfb06cf4.png)

The code works but with an eslint error:
![image](https://user-images.githubusercontent.com/31237954/124490927-4207f880-dde5-11eb-9183-f702338b34bc.png)

**Comment by ota-meshi** [comment]
Please provide the complete source code.  Otherwise, this issue cannot be resolved and will be closed.

**Comment by brenner8023** [comment]
> Please provide the complete source code

Provided

**Comment by ota-meshi** [comment]
I can't try your source code because I can't copy the text from the image.

**Comment by ota-meshi** [comment]
I haven't tried your source code, but I don't think you need to use vue-eslint-parser because vue-eslint-parser is a parser for parsing `.vue`. Also, it is `@typescript-eslint/parser` that actually throws a parse error because `vue-eslint-parser` falls back to `parserOptions.parser` for non-`.vue` source code and parses it, `vue-eslint-parser` can't solve your problem.

I'm not familiar with it because I don't use JSX, but looking at the docs it looks like you should write `v-model={[radioRefValue.value, "value"]}`.

https://github.com/vuejs/jsx-next#directives
